### PR TITLE
Add refresh table interval

### LINKED ISSUE #84: Add interval for refresh_table
Now it's 3600s, I'd like to specify it like `server.refresh_table(interval=30)`

### PR DESCRIPTION
**Pull request by kigawas**
Fixes #84

### CODE REVIEW

**Comment by kigawas** [comment]
```
$ pytest
=============================================================================================== test session starts ================================================================================================
platform darwin -- Python 3.8.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1 -- /Users/li100591/works/py/kademlia/venv/bin/python3
cachedir: .pytest_cache
rootdir: /Users/li100591/works/py/kademlia, configfile: pytest.ini
plugins: cov-2.10.1, asyncio-0.14.0
collected 20 items

kademlia/tests/test_linting.py::TestCodeLinting::test_pylint PASSED                                                                                                                                          [  5%]
kademlia/tests/test_linting.py::TestCodeLinting::test_pep8 PASSED                                                                                                                                            [ 10%]
kademlia/tests/test_node.py::TestNode::test_long_id PASSED                                                                                                                                                   [ 15%]
kademlia/tests/test_node.py::TestNode::test_distance_calculation PASSED                                                                                                                                      [ 20%]
kademlia/tests/test_node.py::TestNodeHeap::test_max_size PASSED                                                                                                                                              [ 25%]
kademlia/tests/test_node.py::TestNodeHeap::test_iteration PASSED                                                                                                                                             [ 30%]
kademlia/tests/test_node.py::TestNodeHeap::test_remove PASSED                                                                                                                                                [ 35%]
kademlia/tests/test_routing.py::TestKBucket::test_split PASSED                                                                                                                                               [ 40%]
kademlia/tests/test_routing.py::TestKBucket::test_split_no_overlap PASSED                                                                                                                                    [ 45%]
kademlia/tests/test_routing.py::TestKBucket::test_add_node PASSED                                                                                                                                            [ 50%]
kademlia/tests/test_routing.py::TestKBucket::test_remove_node PASSED                                                                                                                                         [ 55%]
kademlia/tests/test_routing.py::TestKBucket::test_in_range PASSED                                                                                                                                            [ 60%]
kademlia/tests/test_routing.py::TestKBucket::test_replacement_factor PASSED                                                                                                                                  [ 65%]
kademlia/tests/test_routing.py::TestRoutingTable::test_add_contact PASSED                                                                                                                                    [ 70%]
kademlia/tests/test_routing.py::TestTableTraverser::test_iteration PASSED                                                                                                                                    [ 75%]
kademlia/tests/test_server.py::test_storing PASSED                                                                                                                                                           [ 80%]
kademlia/tests/test_server.py::TestSwappableProtocol::test_default_protocol PASSED                                                                                                                           [ 85%]
kademlia/tests/test_server.py::TestSwappableProtocol::test_custom_protocol PASSED                                                                                                                            [ 90%]
kademlia/tests/test_utils.py::TestUtils::test_digest PASSED                                                                                                                                                  [ 95%]
kademlia/tests/test_utils.py::TestUtils::test_shared_prefix PASSED                                                                                                                                           [100%]

================================================================================================= warnings summary =================================================================================================
kademlia/tests/test_server.py::test_storing
kademlia/tests/test_server.py::test_storing
kademlia/tests/test_server.py::test_storing
kademlia/tests/test_server.py::test_storing
kademlia/tests/test_server.py::test_storing
  /Users/li100591/works/py/kademlia/venv/lib/python3.8/site-packages/rpcudp/protocol.py:83: DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead
    func = asyncio.coroutine(func)

-- Docs: https://docs.pytest.org/en/stable/warnings.html

---------- coverage: platform darwin, python 3.8.6-final-0 -----------
Name                             Stmts   Miss  Cover   Missing
--------------------------------------------------------------
kademlia/__init__.py                 1      0   100%
kademlia/crawling.py                88     13    85%   57, 68, 93, 97-99, 104-107, 118, 124, 143, 150
kademlia/network.py                122     33    73%   56, 89-93, 100, 112-113, 147, 151-152, 162, 178-180, 190, 200-211, 220-227, 239-241
kademlia/node.py                    65      6    91%   39, 82-85, 123
kademlia/protocol.py                85     19    78%   27-28, 32, 61, 77-79, 105-113, 122-124
kademlia/routing.py                130     19    85%   63-64, 69, 77-79, 82, 85, 139-141, 152-153, 169-173, 183, 193
kademlia/storage.py                 54      4    93%   56, 62, 75-76
kademlia/tests/__init__.py           0      0   100%
kademlia/tests/conftest.py          37      0   100%
kademlia/tests/test_linting.py      17      2    88%   18, 26
kademlia/tests/test_node.py         41      0   100%
kademlia/tests/test_routing.py      84      0   100%
kademlia/tests/test_server.py       35      0   100%
kademlia/tests/test_storage.py      22     16    27%   6-8, 11-13, 16-20, 23-27
kademlia/tests/test_utils.py        17      0   100%
kademlia/utils.py                   21      2    90%   40-41
--------------------------------------------------------------
TOTAL                              819    114    86%

========================================================================================== 20 passed, 5 warnings in 7.76s ==========================================================================================
```

**Comment by kigawas** [comment]
pytest: 
```
kademlia/tests/test_linting.py::TestCodeLinting::test_pylint PASSED                                                                                                                                          [  5%]
kademlia/tests/test_linting.py::TestCodeLinting::test_pep8 PASSED                                                                                                                                            [ 10%]
kademlia/tests/test_node.py::TestNode::test_long_id PASSED                                                                                                                                                   [ 15%]
kademlia/tests/test_node.py::TestNode::test_distance_calculation PASSED                                                                                                                                      [ 20%]
kademlia/tests/test_node.py::TestNodeHeap::test_max_size PASSED                                                                                                                                              [ 25%]
kademlia/tests/test_node.py::TestNodeHeap::test_iteration PASSED                                                                                                                                             [ 30%]
kademlia/tests/test_node.py::TestNodeHeap::test_remove PASSED                                                                                                                                                [ 35%]
kademlia/tests/test_routing.py::TestKBucket::test_split PASSED                                                                                                                                               [ 40%]
kademlia/tests/test_routing.py::TestKBucket::test_split_no_overlap PASSED                                                                                                                                    [ 45%]
kademlia/tests/test_routing.py::TestKBucket::test_add_node PASSED                                                                                                                                            [ 50%]
kademlia/tests/test_routing.py::TestKBucket::test_remove_node PASSED                                                                                                                                         [ 55%]
kademlia/tests/test_routing.py::TestKBucket::test_in_range PASSED                                                                                                                                            [ 60%]
kademlia/tests/test_routing.py::TestKBucket::test_replacement_factor PASSED                                                                                                                                  [ 65%]
kademlia/tests/test_routing.py::TestRoutingTable::test_add_contact PASSED                                                                                                                                    [ 70%]
kademlia/tests/test_routing.py::TestTableTraverser::test_iteration PASSED                                                                                                                                    [ 75%]
kademlia/tests/test_server.py::test_storing PASSED                                                                                                                                                           [ 80%]
kademlia/tests/test_server.py::TestSwappableProtocol::test_default_protocol PASSED                                                                                                                           [ 85%]
kademlia/tests/test_server.py::TestSwappableProtocol::test_custom_protocol PASSED                                                                                                                            [ 90%]
kademlia/tests/test_utils.py::TestUtils::test_digest PASSED                                                                                                                                                  [ 95%]
kademlia/tests/test_utils.py::TestUtils::test_shared_prefix PASSED                                                                                                                                           [100%]

```

File: kademlia/network.py [hunk 1]
```diff
@@ -73,11 +73,11 @@ async def listen(self, port, interface='0.0.0.0'):
         # finally, schedule refreshing table
         self.refresh_table()
 
-    def refresh_table(self):
+    def refresh_table(self, interval: float = 3600):
```
Comment (bmuller) [inline kademlia/network.py:76]: There are no other typehints in the code, so would prefer to maintain consistency.

**Review by bmuller — CHANGES_REQUESTED** [review]
Consistency w/ the other code

**Comment by bmuller** [comment]
Thanks @kigawas!

**Pull request merged**